exercicios/para-casa/entregas/beatriz-locatelli/classes/AnimaisExoticos.js

```diff
@@ -0,0 +1,25 @@
+const { HistoricoMedico } = require('./HistoricoMedico');
+
+function AnimaisExoticos(nome, idade, cor, especie, adulto) {
+    this.nome = nome;
+    this.idade = idade;
+    this.cor = cor;
+    this.especie = especie;
+    this.historico = new HistoricoMedico();
+    this.adulto = adulto
+    this.ehAdulto = () => {
+        if (this.especie === 'Hamster' && this.idade >= '3'){
+            return true
+        }
+        if (this.especie === 'Papagaio' && this.idade >= '12'){
```

Comment:
Aqui eu vou deixar uma sugestão, que seria o uso do `else if`. O uso de muitos `if` causa alocação de memória desnecessário. Por exemplo, em caso do animal for Hamster e fosse feito a utilização do `else if`  ele faria apenas a primeira verificação já no caso do uso do if ele faria a primeira e continuaria fazendo as demais verificações mesmo que a primeira fosse a correta.

Reply:
Muito obrigada por essa informação! Vou implementar dessa forma.